cinema/views.py

```diff
@@ -1 +1,67 @@
-# write views here
+from rest_framework import viewsets
+from cinema.models import (
+    Movie,
+    Genre,
+    Actor,
+    CinemaHall,
+    MovieSession,
+)
+from cinema.serializers import (
+    MovieSerializer,
+    ActorSerializer,
+    GenreSerializer,
+    CinemaHallSerializer,
+    MovieSessionSerializer,
+    MovieSessionListSerializer,
+    MovieRetrieveSerializer,
+    MovieSessionRetrieveSerializer, MovieListSerializer,
+)
+
+
+class MovieSessionViewSet(viewsets.ModelViewSet):
+    queryset = MovieSession.objects.all()
+
+    def get_serializer_class(self):
+        if self.action == "list":
+            return MovieSessionListSerializer
+        elif self.action == "retrieve":
+            return MovieSessionRetrieveSerializer
+        return MovieSessionSerializer
```

Comment:
Consider adding type annotations to the `get_serializer_class` method to improve code readability and maintainability, as specified in the checklist.